src/core/diff/strategies/new-unified/__tests__/edit-strategies.test.ts

```diff
@@ -1,3 +1,5 @@
+/// <reference types="jest" />
```

Comment:
It seems like we have to disambiguate between mocha and jest when both sets of globals are installed... in this case VSCode doesn't think `expect` has a second parameter (the timeout) because it's picking up the type from mocha. I'm not sure what the "correct" fix for this is.

Longer term I think the integration tests (and by extension all of the mocha stuff) should live in it's own package like `webview-ui` does (and ideally the packages should be a proper monorepo).